src/domain/CallLog.js

```diff
@@ -182,6 +182,8 @@ export default class CallLog {
   }
 
   isADeclinedCall(session: Session): boolean {
-    return this.destination.extension === session.primaryNumber() && !this.answered;
+    const allNumbers = session.allNumbers();
+    return allNumbers ? allNumbers
+      .filter(number => number === this.destination.extension).length > 0 && !this.answered : false;
```

Comment:
Est-ce que ca pourrait etre : 
`return !this.answered && allNumbers.some(number => number === this.destination.extension);`